app/controllers/v3/packages_controller.rb

```diff
@@ -168,11 +172,13 @@ def create_copy
     unprocessable_app! unless destination_app &&
       permission_queryer.can_read_from_space?(destination_app.space.guid, destination_app.organization.guid)
     unauthorized! unless permission_queryer.can_write_to_space?(destination_app.space.guid)
+    suspended! unless permission_queryer.is_org_active?(destination_app.organization.guid)
 
     source_package = PackageModel.where(guid: hashed_params[:source_guid]).first
     unprocessable_source_package! unless source_package &&
       permission_queryer.can_read_from_space?(source_package.space.guid, source_package.space.organization.guid)
     unauthorized! unless permission_queryer.can_write_to_space?(source_package.space.guid)
+    suspended! unless permission_queryer.is_org_active?(source_package.space.organization.guid)
```

Comment:
Add tests to `spec/request/packages_spec.rb`:
- POST /v3/packages [creation / copying]
- POST /v3/packages/:guid/upload
- PATCH /v3/packages/:guid
- DELETE /v3/packages/:guid